src/sage/tensor/modules/free_module_tensor.py

```diff
@@ -212,6 +212,9 @@ class being:
     from sage.symbolic.expression import Expression
     from sage.tensor.modules.finite_rank_free_module import FiniteRankFreeModule
     from sage.tensor.modules.free_module_basis import FreeModuleBasis
+    from sage.manifolds.differentiable.metric import PseudoRiemannianMetric
+    from sage.manifolds.differentiable.poisson_tensor import PoissonTensorField
+    from sage.manifolds.differentiable.symplectic_form import SymplecticForm
```

Comment:
It's only a dependency for the types, not at runtime. But yes, ideally you would have linear counter parts for metrics, poisson tensors and symplectic forms.

Reply:
You can still run pyright on in it. It will just add a warning that these imports cannot be resolved; but given that 4000 other imports are currently not resolved due to various other issues this shouldn't matter - and we are far away to treat pyright warnings as errors anyway.

Reply:
Thanks, that's a good point, it's fine with me then.